xarray/core/groupby.py

```diff
@@ -22,6 +22,20 @@
 from .variable import IndexVariable, Variable, as_variable
 
 
+def check_reduce_dims(reduce_dims, dimensions):
+
+    if isinstance(reduce_dims, str):
```

Comment:
Technically, should we use `is_scalar` here?

Reply:
:+1: